packages/astro/src/@types/astro.ts

```diff
@@ -1375,7 +1375,6 @@ export interface AstroSettings {
 	tsConfig: TsConfigJson | undefined;
 	tsConfigPath: string | undefined;
 	watchFiles: string[];
-	forceDisableTelemetry: boolean;
```

Comment:
For the code review, you many notice `disableTelemetry`, `forceDisableTelemtry`, etc gets removed. I found these options only exists for tests to pass a value.

I removed it to simplify the code and rely on `process.env.ASTRO_TELEMETRY_DISABLED = true` instead.